wire-ios/Wire-iOS Tests/UserProfile/ProfileViewControllerTests.swift

```diff
@@ -17,9 +17,9 @@
 //
 
 import SnapshotTesting
-import XCTest
 @testable import Wire
 @testable import WireSyncEngineSupport
+import XCTest
```

Comment:
Does this work with the SwiftLint rule to sort imports?

Reply:
When I updated my base branch SwiftLint was complaining these imports were unsorted so I corrected it here.

Reply:
okay I see, interesting. I think one can break it block between testable and normal imports, but it's just formatting.